kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -282,11 +276,14 @@ define([
         const self = this;
         $([Jupyter.events]).on('before_save.Notebook', () => {
             $('#kb-save-btn').find('div.fa-save').addClass('fa-spin');
+            $('#kb-save-btn').prop('disabled', true);
```

Comment:
Prevent spamming the save button now. I don't think it had an effect before (at least, I couldn't get it to have one), but now it can't.